libs/localization/README.md

```diff
@@ -124,28 +125,35 @@ return (
 - Pre-declaring using `defineMessage` or `defineMessages` for later consumption
 
 ```typescript
-const message = defineMessage({
-  id: 'global:title',
-  defaultMessage: 'Default title',
-  description: 'This is a default title in the global namespace',
+const greeting = defineMessage({
+  id: 'global:greeting',
+  defaultMessage: 'Default greeting',
+  description: 'This is a default greeting in the global namespace',
 })
 
 const messages = defineMessages([
-  {
+  header: {
     id: 'global:title',
```

Comment:
Just so it follows your convention, `global:header` and same below

Reply:
I was intentionally making the `key` different from the `id` to emphasize they're independent of each other.

But I notice I used it as `messages.title` which I've now changed to `messages.header` to match with this one.